CONTRIBUTING.md

```diff
@@ -32,12 +32,14 @@ mockrobiota/
         ├── README.md # Description, information, and known issues/notes on dataset usage
         ├── dataset-metadata.tsv # dataset metadata
         ├── greengenes # database name
-        │   └── 13_8 # database version
-        │       ├── database-identifiers.tsv # database identifiers associated with each mock community member (optional)
-        │       └── expected-taxonomy.tsv # per-sample taxonomic abundances
+        │   └── 13-8 # database version
+        │       └── 99-otus # otu % similarity if applicable. If using a database that has not been clustered, use "100-otus"
+        │           ├── database-identifiers.tsv # database identifiers associated with each mock community member (optional)
+        │           └── expected-taxonomy.tsv # per-sample taxonomic abundances
         ├── sample-metadata.tsv # QIIME-compatible mapping file
         └── source
-            └── taxonomy.tsv # per-sample taxonomic abundances
+            ├── taxonomy.tsv # per-sample taxonomic abundances
+            ├── expected-sequences.fasta # expected SHORT sequences, e.g., full-length-16S-expected-sequences.fasta
```

Comment:
I think `e.g., full-length-16S-expected-sequences.fasta` is confusing, because that couldn't be the file name (right?). Maybe instead: *e.g., full-length 16S sequences*